stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -389,6 +420,42 @@ public class MessageListView : ConstraintLayout {
         setMessageListItemAdapter(adapter)
     }
 
+    /**
+     * @param view will be added to the view hierarchy of [ChannelsView] and managed by it.
+     * The view should not be added to another [ViewGroup] instance elsewhere.
+     * @param layoutParams defines how the view will be situated inside it's container ViewGroup.
+     */
+    public fun setLoadingView(view: View, layoutParams: FrameLayout.LayoutParams = defaultChildLayoutParams()) {
```

Comment:
Why not add a way for the user to pass their own view using XML instead of passing the view programmatically? I think it is easier to create the layout using XML than using code.

Reply:
It would be a nice feature. I'd add this in the next iteration.